docs/src/config-file.md

```diff
@@ -606,11 +606,19 @@ DocSearch to index all your content.
 
 You can override the user agent used to crawl your website. By default, this
 value is:
-`'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/70.0.3538.110 Safari/537.36'`
+
+    Algolia DocSearch Crawler
```

Comment:
```suggestion
     `Algolia DocSearch Crawler`
```

Reply:
I think the current formatting will put the UA in its own code block. Wrapping in backticks should correctly put it inline.

Reply:
I let you merge if you agree on that